web/tests/Makefile

```diff
@@ -44,97 +47,96 @@ PYLINT_TEST_CMD = pylint ./bin/codechecker-web-version \
   --disable=all \
   --enable=logging-format-interpolation,old-style-class
 
-pylint: venv_dev
-	$(ACTIVATE_DEV_VENV) && $(PYLINT_TEST_CMD)
+pylint:
+	 $(PYLINT_TEST_CMD)
 
+pylint_in_env:
+	$(ACTIVATE_DEV_VENV) && $(PYLINT_TEST_CMD)
+ 
 CODECHECKER_CMD = $(BUILD_DIR)/CodeChecker/bin/CodeChecker
 SHUTDOWN_SERVER_CMD = echo "Shutting down server..."; \
-    HOME="$(CC_TEST_WORKSPACE_ROOT)" ${CODECHECKER_CMD} server -l; \
-    HOME="$(CC_TEST_WORKSPACE_ROOT)" ${CODECHECKER_CMD} server \
-			--config-directory $(CC_TEST_WORKSPACE_ROOT) \
-	    --port `cat "$(CC_TEST_WORKSPACE_ROOT)/serverport"` --stop; \
-    rm -f "$(CC_TEST_WORKSPACE_ROOT)/serverport"; \
-    HOME="$(CC_TEST_WORKSPACE_ROOT)" ${CODECHECKER_CMD} server -l
+  HOME="$(CC_TEST_WORKSPACE_ROOT)" ${CODECHECKER_CMD} server -l; \
+  HOME="$(CC_TEST_WORKSPACE_ROOT)" ${CODECHECKER_CMD} server \
+  --config-directory $(CC_TEST_WORKSPACE_ROOT) \
```

Comment:
These two lines can be indented a little bit more (with 2 more spaces) to see that these are the parameters of the CodeChecker server command.